django_extensions/management/commands/show_templatetags.py

```diff
@@ -93,14 +97,22 @@ def handle(self, *args, **options):
                 app_labeled = False
                 for taglib in tag_files:
                     try:
-                        lib = get_library(taglib)
+                        try:
+                            lib = get_installed_libraries()[taglib]
```

Comment:
@maryokhin there's just one failing test remaining on this branch and it has to do with the show_templatetags command here. I've made django-1.9 compatible code here, with the deprecation of `get_library`, and now the issue is that the `django_extensions` template tag library doesn't seem to be loaded during testing in django1.9 here. Maybe someone can fix the problem? Otherwise I will look again later.